src/main/java/org/opentripplanner/routing/graph/SerializedGraphObject.java

```diff
@@ -50,9 +56,60 @@
 
     private final Collection<Edge> edges;
 
-    public SerializedGraphObject(Graph graph) {
+    /** The config JSON used to build this graph. Allows checking whether the configuration has changed. */
+    public final BuildConfig buildConfig;
+
+    /** Embed a router configuration inside the graph, for starting up with a single file. */
+    public final RouterConfig routerConfig;
```

Comment:
We should discuss whether we want to allow embedding configuration at all.

Reply:
We discussed and decided embedding configuration is useful, but realized some router-config entries are probably used during graph build. This is probably unavoidable, so the distinction between the two configuration files is lost. We should probably merge them.

Reply:
I added a new issue for this: https://github.com/opentripplanner/OpenTripPlanner/issues/3020